.azure-pipelines/test_plan.py

```diff
@@ -188,13 +188,18 @@ def get_token(self):
             "client_secret": self.client_secret,
             "scope": get_scope(self.url)
         }
-        try:
-            resp = requests.post(token_url, headers=headers, data=payload, timeout=10).json()
-            self._token = resp["access_token"]
-            self._token_generate_time = datetime.utcnow()
-            return self._token
-        except Exception as exception:
-            raise Exception("Get token failed with exception: {}".format(repr(exception)))
+        max_attempts = 3
+        for attempt in range(max_attempts):
```

Comment:
If we increase attempt on line 201, the loop here should be while(attempt < max_attpemts), right?

Reply:
Fixed